### PR TITLE
feat: post comment to a blog

### PR DESCRIPTION
**Pull request by Okunola11**
I added the post comment to blog endpoint to the route of the project at `/api/v1/blogs/{blog_id}/comments`. The endpoint accepts the `post` method.

## Description

This endpoint adds comments to a blog post. The user must be authenticated and the content of the comment cannot be null or empty. 

## Related Issue (Link to issue ticket)

The issue for this pull request can be found [here](https://github.com/hngprojects/hng_boilerplate_python_fastapi_web/issues/504).

## Motivation and Context

This endpoint adds comments to blog posts, helping authors get feedback on the thoughts of their users. The request `blog_id` must be of a blog which exists in the database. Any other id get a `404` error.

## How Has This Been Tested?

The endpoint has been tested. The unit tests are in the file `/tests/v1/blogs/test_add_comment.py`.
- [x] Data is added appropriately to the database.
- [x] Appropriate response and status code is returned. 
- [x] Users must be authenticated.
- [x] Non existing `blog_id` will throw a `404` error. 
- [x] Empty comment content will throw a `422` error. 

The test can be run by: ` PYTHONPATH=. pytest /tests/v1/blogs/test_add_comment.py `.

### Affected areas of the code
- I added a `comment.py` file to the route, service and schema folders. The files outline the model for my response data and logic for the endpoint respectively. 
- I added a test file at `/tests/v1/blog/test_add_comment.py`.

### Success Response
Screenshot from 2024-07-29 06-32-51

### No content in request body
Screenshot from 2024-07-29 06-33-07

### Unauthenticated user response
Screenshot from 2024-07-29 06-33-26

## Types of changes

- [x] New feature (non-breaking change which adds functionality)    ​

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the readme accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/services/comment.py
Comment (joboy-dev) [inline api/v1/services/comment.py:1]: Nice

**Review by joboy-dev — DISMISSED** [review]

**Comment by joboy-dev** [comment]
Fix merge conflict

File: api/v1/services/comment.py
Comment (Okunola11) [reply api/v1/services/comment.py:1]: Thanks boss.

**Comment by Okunola11** [comment]
It has been fixed @joboy-dev

**Comment by joboy-dev** [comment]
Fix merge conflict

**Review by joboy-dev — DISMISSED** [review]

**Review by Takinnuoye5 — APPROVED** [review]
approved

**Comment by joboy-dev** [comment]
Do not merge

**Review by joboy-dev — APPROVED** [review]

**Review by Takinnuoye5 — APPROVED** [review]
approved

**Pull request merged**